go.sum

```diff
@@ -193,6 +193,7 @@ github.com/go-kit/kit v0.8.0/go.mod h1:xBxKIO96dXMWWy0MnWVtmwkA9/13aqxPnvrjFYMA2
 github.com/go-kit/kit v0.9.0/go.mod h1:xBxKIO96dXMWWy0MnWVtmwkA9/13aqxPnvrjFYMA2as=
 github.com/go-ldap/ldap v3.0.2+incompatible/go.mod h1:qfd9rJvER9Q0/D/Sqn1DfHRoBp40uXYvFoEVrNEPqRc=
 github.com/go-logfmt/logfmt v0.3.0/go.mod h1:Qt1PoO58o5twSAckw1HlFXLmHsOX5/0LbT9GBnD5lWE=
+github.com/go-logfmt/logfmt v0.4.0 h1:MP4Eh7ZCb31lleYCFuwm0oe4/YGak+5l1vA2NOE80nA=
```

Comment:
Any idea why this got updated?

Reply:
Not sure 🤨

go mod why doesn't seem helpful:

```sh
% go mod why github.com/go-logfmt/logfmt
# github.com/go-logfmt/logfmt
(main module does not need package github.com/go-logfmt/logfmt)
% go mod why -m github.com/go-logfmt/logfmt
# github.com/go-logfmt/logfmt
(main module does not need module github.com/go-logfmt/logfmt)
```

Taking a look at the dependencies' go.mod:

```sh
% go mod vendor && cd vendor
% rg github.com/go-logfmt/logfmt
github.com/munnerz/crd-schema-fuzz/go.sum
69:github.com/go-logfmt/logfmt v0.3.0/go.mod h1:Qt1PoO58o5twSAckw1HlFXLmHsOX5/0LbT9GBnD5lWE=

github.com/spf13/cobra/go.sum
27:github.com/go-logfmt/logfmt v0.3.0/go.mod h1:Qt1PoO58o5twSAckw1HlFXLmHsOX5/0LbT9GBnD5lWE=
28:github.com/go-logfmt/logfmt v0.4.0/go.mod h1:3RMwSq7FuexP4Kalkev3ejPJsZTpXXBr9+V4qmtdjCk=

github.com/Venafi/vcert/v4/go.sum
47:github.com/go-logfmt/logfmt v0.3.0/go.mod h1:Qt1PoO58o5twSAckw1HlFXLmHsOX5/0LbT9GBnD5lWE=
48:github.com/go-logfmt/logfmt v0.4.0/go.mod h1:3RMwSq7FuexP4Kalkev3ejPJsZTpXXBr9+V4qmtdjCk=

go.uber.org/zap/glide.lock
27:- name: github.com/go-logfmt/logfmt

sigs.k8s.io/controller-runtime/go.sum
91:github.com/go-logfmt/logfmt v0.3.0/go.mod h1:Qt1PoO58o5twSAckw1HlFXLmHsOX5/0LbT9GBnD5lWE=
```

Looks like cobra and vcert rely on v0.4.0... but not sure why both v0.3.0 and v0.4.0 are both required :(